fontforgeexe/uiutil.c

```diff
@@ -692,6 +692,7 @@ struct ui_interface gdraw_ui_interface = {
 
     UI_TTFNameIds,
     UI_MSLangString,
-    (int (*)(void)) Ps_StrokeFlagsDlg
+    _ImportParamsDlg,
```

Comment:
Why are these underscored?

Reply:
`ImportParamsDlg` and `ExportParamsDlg` are the defines that redirect through `ui_interface`. These are the UI-side implementations.

Reply:
Yeah, I mean why isn't the naming scheme consistent with the rest of the other functions in this list?

Reply:
I guess that's an idea but the other functions starting with `UI_` are defined in this file and the one I'm replacing that was not defined in this file didn't follow that convention.